activiti-cloud-services-common-security-keycloak/src/main/java/org/activiti/cloud/services/common/security/keycloak/config/CommonSecurityAutoConfiguration.java

```diff
@@ -51,11 +56,27 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
         auth.authenticationProvider(keycloakAuthenticationProvider);
     }
 
+    @Override
     protected KeycloakAuthenticationProvider keycloakAuthenticationProvider() {
-        return new KeycloakAuthenticationProvider();
+        return keycloakAuthenticationProvider;
     }
-
+    
+    @Bean
+    @Primary
+    @ConditionalOnMissingBean
```

Comment:
I guess that `@ConditionalOnMissingBean` is not needed if it's `@Primary`, right? BTW, do you why we need a `@Primary` here?

Reply:
@erdemedeiros You may be right. `@Primary` was needed because there were 2 candidates caused by `@ComponentScan`. I think it is a good idea to remove `@Primary`. I will try it when the propagation reaches this PR.